### PR TITLE
Fix linter errors & numpy dependency workaround

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

## Category:

**Other** (formatting)

## Description:

- Adjusts formatting to fix issues found by the linter

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

**Review by JanuszL — APPROVED** [review]

File: qa/setup_packages.py [hunk 1]
```diff
@@ -210,9 +210,9 @@ def get_install_string(self, idx, cuda_version=None):
             `cuda_version`: str, optional, default = None
                 Cuda version used for this query
         """
-        version = version=self.get_version(idx, cuda_version)
+        version = version = self.get_version(idx, cuda_version)
```
Comment (mzient) [inline qa/setup_packages.py:213]:
```suggestion
        version = self.get_version(idx, cuda_version)
```

**Review by mzient — APPROVED** [review]

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Review by mzient — APPROVED** [review]

**Comment by jantonguirao** [comment]
!build

**Pull request merged**